FortnitePorting/Plugins/Blender/import_task.py

```diff
@@ -574,6 +600,128 @@ def get_meta(search_props):
             case _:
                 meta = {}
 
+        # fetch pose data
+        meta.update(get_meta(["PoseData", "ReferencePose"]))
+        if (pose_data := meta.get("PoseData")):
+            is_head = mesh_type == "Head"
+            shape_keys = imported_mesh.data.shape_keys
+            armature: bpy.types.Object = imported_object
+            original_mode = bpy.context.active_object.mode
+            try:
+                bpy.ops.object.mode_set(mode='OBJECT')
+                armature_modifier: bpy.types.ArmatureModifier = first(imported_mesh.modifiers, lambda mod: mod.type == "ARMATURE")
+
+                # Grab reference pose data
+                reference_pose = meta.get("ReferencePose", dict())
+
+                # Check how many bones are scaled in the reference pose
+                face_attach_scale = Vector((1, 1, 1))
+                for entry in reference_pose:
+                    scale = make_vector(entry.get('Scale'))
+                    if scale != Vector((1, 1, 1)):
+                        if entry.get('BoneName', '').casefold() == 'faceattach':
```

Comment:
the bone name "FACIAL_C_FacialRoot" is used for metahuman face skins, does this need to be added here too?

Reply:
metahuman skins are newer skins like lorenzo, rian, the ageless, etc.